lib/models/mirror.js

```diff
@@ -15,9 +16,13 @@ var Mirror = new mongoose.Schema({
   },
   contact: {
     type: String,
-    ref: 'Contact'
+    ref: 'Contact',
+    required: true
```

Comment:
Okay, manually tested, and there are not any issues. Any mirror that doesn't have a contact, *for some reason*, will yield an error such as this, on the cli side that doesn't handle the situation: https://github.com/Storj/core-cli/issues/82